main/dpkg/APKBUILD

```diff
@@ -1,17 +1,16 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=dpkg
-pkgver=1.19.2
-pkgrel=1
+pkgver=1.19.4
+pkgrel=0
 pkgdesc="The Debian Package Manager"
 url="http://packages.debian.org/dpkg"
 arch="all"
-license="GPL"
-depends="xz"
+license="GPL-2.0-or-later BSD-2-Clause"
+depends="xz tar"
```

Comment:
i'd probably sort the depends alphabetically, and put the tar before the xz...

Reply:
I forgot first time around. Done now.